python/lsst/validate/drp/matchreduce.py

```diff
@@ -190,6 +194,17 @@ def _loadAndMatchCatalogs(repo, dataIds, matchRadius,
                                        'PSF magnitude'))
     mapper.addOutputField(Field[float]('base_PsfFlux_magErr',
                                        'PSF magnitude uncertainty'))
+    if not skipGalaxies:
```

Comment:
Can you re-phrase this as an interactive check to see if the necessary columns are there in the data?